slices/gearman-job-server.yaml

```diff
@@ -0,0 +1,28 @@
+package: gearman-job-server
+
+essential:
+  - gearman-job-server_copyright
+
+slices:
+  libs:
```

Comment:
So for gearman-job-server, let's create the following slices:
1) bins:
-- Should just have `/usr/sbin/gearmand`.
2) services:
-- Should just have `/usr/lib/systemd/system/gearman-job-server.service`.

All the other files can be omitted from the slices.